From 4460c87a37c193d3618fb2d869e7e95ba5591606 Mon Sep 17 00:00:00 2001
From: Peter Geis <pgwipeout@gmail.com>
Date: Sat, 26 Feb 2022 13:41:44 -0500
Subject: [PATCH 04/99] usb: dwc3: add rk3568 dwc3 support

The rk3568 dwc3 controller is backwards compatible with the rk3399 dwc3
controller.
Add support for it to the dwc3-of-simple driver.

Signed-off-by: Peter Geis <pgwipeout@gmail.com>
---
 drivers/usb/dwc3/dwc3-of-simple.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/usb/dwc3/dwc3-of-simple.c b/drivers/usb/dwc3/dwc3-of-simple.c
index 57ba9427ad74..d9d1c5bfac3f 100644
--- a/drivers/usb/dwc3/dwc3-of-simple.c
+++ b/drivers/usb/dwc3/dwc3-of-simple.c
@@ -195,6 +195,10 @@ static const struct of_device_id of_dwc3_simple_match[] = {
 		.compatible = "rockchip,rk3399-dwc3",
 		.data = &dwc3_of_simple_data_rk3399,
 	},
+	{
+		.compatible = "rockchip,rk3568-dwc3",
+		.data = &dwc3_of_simple_data_rk3399,
+	},
 	{
 		.compatible = "sprd,sc9860-dwc3",
 	},
-- 
2.37.3

